aikau/src/main/resources/alfresco/layout/AlfStackContainer.js

```diff
@@ -0,0 +1,558 @@
+ *             name: "alfresco/logo/Logo",
+ *             title: "Pane with Alfresco Logo",
+ *             selected
+ *             config: {
+ *                logoClasses: "alfresco-logo-only"
+ *             }
+ *          }
+ *       ]
+ *    }
+ * }
+ *
+ * @example <caption>Example publication to add a new pane (based on previous example topic):</caption>
+ * {
+ *    publishTopic: "ADD_PANE_TOPIC",
+ *    publishPayload: {
+ *       widgets: [
+ *          {
+ *             name: "alfresco/html/Label",
+ *             title: "New",
+ *             closable: true,
+ *             selected: true,
+ *             config: {
+ *                label: "This pane was dynamically added"
+ *             }
+ *          }
+ *       ]
+ *    }
+ * }
+ *
+ * @example <caption>Define a paneSelectionHashVar:</caption>
+ * {
+ *    name: "alfresco/layout/AlfStackContainer",
+ *    config: {
+ *       paneSelectionHashVar: "view",
+ *       widgets: [
+ *          {
+ *             id: "PANE1",
+ *             name: "alfresco/logo/Logo",
+ *             title: "Pane1",
+ *             selected
+ *             config: {
+ *                logoClasses: "alfresco-logo-only"
+ *             }
+ *          },
+ *          {
+ *             id: "PANE2",
+ *             name: "alfresco/logo/Logo",
+ *             title: "Pane2",
+ *             selected: true,
+ *             config: {
+ *                logoClasses: "surf-logo-large"
+ *             }
+ *          }
+ *       ]
+ *    }
+ * }
+ * <p>If the hash was changed to #view=Pane2, PANE2 will be displayed</p>
+ * 
+ * @module alfresco/layout/AlfStackContainer
+ * @extends external:dijit/_WidgetBase
+ * @mixes external:dojo/_TemplatedMixin
+ * @mixes module:alfresco/core/Core
+ * @mixes module:alfresco/core/CoreWidgetProcessing
+ * @author Richard Smith
+ */
+define(["dojo/_base/declare",
+        "dijit/_WidgetBase", 
+        "dijit/_TemplatedMixin",
+        "dojo/text!./templates/AlfStackContainer.html",
+        "alfresco/core/Core",
+        "alfresco/core/CoreWidgetProcessing",
+        "alfresco/documentlibrary/_AlfHashMixin",
+        "dijit/layout/StackContainer",
+        "dijit/layout/ContentPane",
+        "dojo/dom-construct",
+        "dojo/_base/lang",
+        "dojo/_base/array",
+        "dojo/hash",
+        "dojo/io-query"], 
+        function(declare, _WidgetBase, _TemplatedMixin, template, AlfCore, CoreWidgetProcessing, _AlfHashMixin, 
+                  StackContainer, ContentPane, domConstruct, lang, array, hash, ioQuery) {
+   
+   return declare([_WidgetBase, _TemplatedMixin, AlfCore, CoreWidgetProcessing, _AlfHashMixin], {
+      
+      /**
+       * The HTML template to use for the widget
+       * 
+       * @instance
+       * @type {string}
+       */
+      templateString: template,
+      
+      /**
+       * This will hold a reference to the stack container widget.
+       *
+       * @instance
+       * @type {object}
+       * @default null
```

Comment:
Do not need to specify a value when using the `@default` keyword, when describing `simple` variables. See the JSDoc website for further details.